### PR TITLE
LTP: Fixed fallocate04 and fallocate05 testcases

### PR DESCRIPTION
**Pull request by vamsikrishna935**
Tests were failing with Kernel panic (out of memory) in loop device.
loop device filesystem is having a limit of 32MB
So modified the tests using root filesystem.

### CODE REVIEW

**Comment by davidchisnall** [comment]
I think this patch is fine, but please can we track the tests that are failing because we don't have enough kernel memory centrally?  These should all be revisited after #597 is fixed.

**Review by davidchisnall — APPROVED** [review]

File: tests/ltp/patches/ltp_fallocate_fallocate04_fix.patch [hunk 1]
```diff
@@ -0,0 +1,44 @@
++ Tesst Failure reason:
```
Comment (hukoyu) [inline tests/ltp/patches/ltp_fallocate_fallocate04_fix.patch]: Can you change the patch file name to fallocate04.patch
You can run 
git mv ltp_fallocate_fallocate04_fix.patch  fallocate04.patch

File: tests/ltp/patches/ltp_fallocate_fallocate05_fix.patch [hunk 2]
```diff
@@ -0,0 +1,66 @@
++ Test failure reason:
```
Comment (hukoyu) [inline tests/ltp/patches/ltp_fallocate_fallocate05_fix.patch]: Can you change the patch file name to fallocate05.patch
You can run 
git mv ltp_fallocate_fallocate05_fix.patch  fallocate05.patch

**Review by hukoyu — CHANGES_REQUESTED** [review]
@vamsikrishna935 please change patch file names as suggested. Afte rthat we can merge this PR.

**Review by hukoyu — APPROVED** [review]
LGTM

**Pull request merged**